src/index.ts

```diff
@@ -8,6 +8,8 @@ import { getLinks } from './links';
 const finalhandler = require('finalhandler');
 const serveStatic = require('serve-static');
 
+declare type url = string;
```

Comment:
At a glance, I'm not sure why we'd do that.  There is a `Url` type in node core thats _very_ different, and I'm worried that folks would confuse this type with that type.  Why doesn't `string` work?

Reply:
I didn't think of the "Url" potential confusion. Good point. My thought is this is just a small step forward to more intelligent types. It just provides a hint that not all strings are valid, so if someone were to do something like crawl("http foo") in the future you potentially throw at dev/compile time instead of runtime. Very much a nit that I don't feel strongly for. Just something I do elsewhere (narrow all the general types) because at the least can help make things more self documenting
.